File: integration-tests/maven/src/test/resources-filtered/projects/test-tests-in-project-with-junit-properties-file/src/test/java/org/acme/HelloResourceTest.java
package org.acme;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.is;

import org.junit.jupiter.api.Test;

import io.quarkus.test.junit.QuarkusTest;

@QuarkusTest
public class HelloResourceTest {

    @Test
    public void testHelloEndpoint() {
        given()
                .when()
                .get("/app/hello")
                .then()
                .statusCode(200)
                .body(is("Hello from Quarkus REST via config"));
    }

}

File: integration-tests/kotlin/src/test/resources/projects/classic-kotlin/src/test/kotlin/org/acme/HelloResourceTest.kt
package org.acme

import io.quarkus.test.junit.QuarkusTest
import org.junit.jupiter.api.Test

import io.restassured.RestAssured.given
import org.hamcrest.CoreMatchers.`is`

@QuarkusTest
class HelloResourceTest {

    @Test
    fun testHelloEndpoint() {
        given()
                .`when`().get("/app/hello")
                .then()
                .statusCode(200)
                .body(`is`("hello"))
    }

}
